keps/sig-ui/NNNN-kep-template/README.md

```diff
@@ -0,0 +1,810 @@
+* Is built on a very common technology stack (Typescript, React, Material UI, Vite, Go)
+* Has more advanced features (multi-cluster, graph view, powerful plugin system, 3rd party plugins)
+* Addresses many more use cases (see below)
+
+
+### Goals
+
+With this proposal we aim to provide the Kubernetes project with the following.
+* A tool to more easily onboard new and beginner-level users
+* A UI that serves as a base for vendors to build upon
+* A more modern UI developer environment
+* A more feature-rich UI
+* A desktop user experience
+* An integration point for other CNCF and external projects, especially those without a UI focus
+
+
+### Non-Goals
+
+This proposal does not propose to force people who are using the current Dashboard to move to Headlamp.
+
+## Proposal
+
+The proposal is to ship the Headlamp desktop experience as a new Kubernetes Desktop and the Headlamp in-cluster UI become the next generation of the Kubernetes Dashboard.
+
+Effectively this entails moving the Headlamp project under the Kubernetes GitHub organization and under the auspices of Kubernetes SIG UI. To be specific, we would propose that the Headlamp project be moved to a repo under the Kubernetes project; something to the effect of kubernetes/ui.
+The current repo kubernetes/dashboard would remain but include a note about the existence of the other repo. This would allow for users to compare the projects and make an informed decision about which works best for their use case. Over time, we can measure adoption trends and make decisions based on that data.
+
+The Headlamp branding would be phased out.
+
+### User Stories (Optional)
+
+One of Headlamp's strengths is that it was designed to both deliver a great end-user experience and be a platform for others to build upon and integrate with. This is demonstrated by the wide range of user stories it enables  and one of the reasons we believe it can become a hub for the community.
+
+_**Note:** Using “Headlamp” in the following examples to avoid confusion._
+
+#### Story 1
+
+As an end user I want to get an overview of my cluster. I start the Headlamp Desktop app which picks up my clusters from my kubeconfig and I’m now able to explore my clusters.
+
+#### Story 2
+
+As a new Kubernetes user I want to learn more about Kubernetes. I use the Headlamp Desktop app to guide me in installing a local kubernetes cluster using minikube. Once my local cluster is installed I’m presented with a tutorial plugin which guides me through the various aspects of the UI and Kubernetes concepts.
+_**Note:** We have the minikube installer plugin in POC but the tutorial plugin is not something we currently have._
+
+#### Story 3
+
+As a cluster admin I was to provide a user interface for my users. I choose which plugins I want to include for my users and deploy Headlamp to my cluster. I have peace of mind knowing that Headlamp respects the RBAC rules that each user has.
```

Comment:
If I'm understanding this properly, it took me a while. I had to read this, the later section on auth, and re-read this user story to understand it. I think the user story should stand on itself.

It seems to me, IIUC, that this usre story tries to mention these things:
 * I want to provide a UI for users and select what headlamp features can be used (or similar, what you want to do with plugins)
 * I want to provide a UI that is not a security nightmare that needs to run with a super-privileged service account
 * I want to provide a UI that only allows users to do what their RBAC permissions allow.

Am I understanding it right?

Maybe splitting it in more user stories would be simpler.

And maybe that would also allow to remove the "reference text" later explaining how auth work? It's not easy to get it if you need to read this, read the auth text below and then re-read it to realize the subtle things this implies.